leftwm/doc/leftwm.1

```diff
@@ -217,36 +217,70 @@ New windows will be created in the workspace:
 .IP "-"
 .B  Containing the cursor
 when unset (\f[C]None\f[R]), \f[C]Some(true)\f[R], or when the cursor is in \f[C]Sloppy\f[R] mode
-.IP "-" 
+.IP "-"
 .B Which is focused
 when set to \f[C]Some(false)\f[R]
 .PP
 Default: \f[C]create_follows_cursor = None\f[R]
 .SS Layouts
 .PP
-Leftwm supports variety of layouts, which define the way that windows are tiled in the workspace
+Leftwm supports a variety of user definable layouts. Layouts define the way that windows are tiled in the workspace.
+.PP
+The layouts considered by leftwm can be customized in the
+.B config.ron
+file by editing the
+.B layouts
+and the
+.B layout_definitions
+entries.
+.PP
+The layouts in the
+.B layouts
+list are those accessible when switching layouts. For an example with a single layout:
+.PP
+.IP
+.nf
+\f[C]
+layouts = [ "MyMainAndVertStack" ]
+\f[R]
+.fi
+.PP
+Each list in the
+.B layouts
+list must have a corresponding definition in the
+.B layout_definitions
+entry. For example:
 .PP
-Default layouts:
 .IP
 .nf
 \f[C]
-layouts: [
-    MainAndDeck,
-    MainAndVertStack,
-    MainAndHorizontalStack,
-    GridHorizontal,
-    EvenHorizontal,
-    EvenVertical,
-    Fibonacci,
-    CenterMain,
-    CenterMainBalanced,
-    CenterMainFluid,
-    Monocle,
-    RightWiderLeftStack,
-    LeftWiderRightStack,
+layout_definitions = [(
+  name: "MyMainAndVertStack",
+  flip: None,
+  rotate: North,
+  reserve: ReserveAndCenter,
+  columns: (
+    flip: None,
+    rotate: North,
+    main: (count: 1, size: 0.8, flip: None, rotate: North, split: None),
+    stack: (flip: None, rotate: North, split: Horizontal),
+    second_stack: None)
+  ),
 ]
```

Comment:
I think the number of parentheses is OK, although perhaps their placement is awkward.

Reply:
This line, there should only be a comma, right?
```
second_stack: None,
```